x/lockdrop/client/cli/tx.go

```diff
@@ -0,0 +1,162 @@
+package cli
+
+import (
+	"fmt"
+
+	"github.com/cosmos/cosmos-sdk/client/tx"
+
+	"github.com/spf13/cobra"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	govcli "github.com/cosmos/cosmos-sdk/x/gov/client/cli"
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types/v1beta1"
+
+	"github.com/osmosis-labs/osmosis/v15/osmoutils"
+
+	"github.com/dymensionxyz/dymension/x/lockdrop/types"
+)
+
+func NewCmdSubmitUpdateLockdropProposal() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "update-lockdrop [gaugeIds] [weights]",
+		Args:  cobra.ExactArgs(2),
+		Short: "Submit an update to the records for pool incentives",
```

Comment:
How does the gaugeIDS and weights are supposed to be passed? Comma separated or with spaces?